DataFormats/Detectors/TRD/src/Tracklet64.cxx

```diff
@@ -19,6 +20,37 @@ namespace o2
 namespace trd
 {
 
+using namespace constants;
+
+float Tracklet64::getY() const
+{
+  int padLocalBin = getPosition();
+  int padLocal = 0;
+  if (padLocalBin & (1 << (NBITSTRKLPOS - 1))) {
+    padLocal = -((~(padLocalBin - 1)) & ((1 << NBITSTRKLPOS) - 1));
+  } else {
+    padLocal = padLocalBin & ((1 << NBITSTRKLPOS) - 1);
+  }
+  int mcmCol = (getMCM() % NMCMROBINCOL) + NMCMROBINCOL * (getROB() % 2);
+  float offset = -63.f + ((float)NCOLMCM) * mcmCol;
+  float padWidth = 0.635f + 0.03f * (getDetector() % NLAYER);
+  return (offset + padLocal * GRANULARITYTRKLPOS) * padWidth;
+}
```

Comment:
I know I'm a bit late with my comment, but I think here we need to make sure that we have a well defined way of converting a tracklet from row/pad/timebin to global coordinates.

This conversion does not take alignment into account. The same applies for getDy, where we even need to take v_drift and ExB into account. 

My idea was that a separate class would generate "calibrated tracklets", where this is implemented (and tested) once. Let's this discuss in today's meeting.